app/controllers/assignments_controller.rb

```diff
@@ -281,6 +281,29 @@ def recreate_grades
                   notice: "#{pluralize(count, 'grade')} created"
   end
 
+  def update_section_toggles
+    toggle_params = params.permit(:assignment_id, :state, :submission_enabled_toggle_id)
+    toggle = SubmissionEnabledToggle.where(id: toggle_params[:submission_enabled_toggle_id]).first
+    if toggle.nil?
+      render json: {}, status: 404
```

Comment:
I've had troubles sending back an empty JSON object.  So I send back `json: {not_found: true}` or something silly like that...  It helps ensure that I really did get a response from the controller, rather than wondering if the connection got reset or something.